### ISSUE TITLE
Create benchmark for ping pong using REST and the compare using client proxy access

### ISSUE THREAD

**Issue by RichardHightower** [advantageous/qbit#77]
Use Spring MVC REST to create a single end point called ping. 

```java

        @RequestMapping(method = RequestMethod.POST)
        public String ping(String ping) {
            callCount++;
            return ping + " pong";
        }

        @RequestMapping(method = RequestMethod.GET)
        public String count() {
            return callCount;
        }

```

Do the same in qbit.

Compare them with wrk, and JMeter or Gatling. 

Now create a client proxy using Spring REST client.

See 22.10 Accessing RESTful services on the Client
http://docs.spring.io/spring/docs/current/spring-framework-reference/html/remoting.html

Do the same in qbit with the client proxy.

Use a test like io.advantageous.qbit.vertx.http.LoadTestingTest to compare throughput of the two client proxies.

**Comment by revelfire** [comment]
Why is this closed? I want to know what happens!!!

**Comment by RichardHightower** [comment]
I created it. Qbit smoked the competition by a large margin.

On Thursday, January 29, 2015, revelfire <notifications@github.com> wrote:

> Why is this closed? I want to know what happens!!!
> On Jan 29, 2015 4:49 PM, "Richard Hightower" <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
>
> > Closed #77 <https://github.com/advantageous/qbit/issues/77>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/advantageous/qbit/issues/77#event-227252091>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/advantageous/qbit/issues/77#issuecomment-72144230>.
>


-- 
*Rick Hightower*
(415) 968-9037
Profile <http://www.google.com/profiles/RichardHightower>
